### ISSUE TITLE
CONSUMER: Should block ACK/REJECT if is in Preview mode

### ISSUE THREAD

**Issue by dshvedchenko** [instaclick/PDI-Plugin-Step-AMQP#18]
1. either not ack/reject always
2. or have option for Developer , to block ack/reject for Preview

**Comment by jcart** [comment]
What about queues that have require ack disabled?

**Comment by dshvedchenko** [comment]
We have option REQUEQUE , when TRUE, disables ack at all

**Comment by jcart** [comment]
Right. This is the obvious solution but you have to consider that this is not deterministic -- nor is there probably a way to accomplish it without ack. The risk should be identified somehow.

**Comment by dshvedchenko** [comment]
NOTE:  at plugin class we have ability to detect Preview Mode : getTrans().isPreview().
for Preview mode we will always requeue messages.
